src/universal/components/Dashboard/DashSearchControl.js

```diff
@@ -1,35 +1,55 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import withStyles from 'universal/styles/withStyles';
-import {css} from 'aphrodite-local-styles/no-important';
+import FontAwesome from 'react-fontawesome';
+import makePlaceholderStyles from 'universal/styles/helpers/makePlaceholderStyles';
+import ui from 'universal/styles/ui';
+import appTheme from 'universal/styles/theme/appTheme';
+import styled from 'react-emotion';
+
+const DashSearch = styled('div')({
+  alignItems: 'center',
+  display: 'flex',
+  flex: 1
+});
+
+const DashSearchIcon = styled(FontAwesome)({
+  color: appTheme.palette.dark60l,
+  fontSize: ui.iconSize,
+  marginRight: '.5rem'
+});
+
+const DashSearchInput = styled('input')({
+  appearance: 'none',
+  display: 'block',
+  border: 0,
+  fontSize: '.875rem',
+  outline: 'none',
+  padding: 0,
+  width: '100%',
```

Comment:
this seems a little _too_ big. maybe have it take up as much as the placeholder? i can't imagine more than 3 words going in here

Reply:
is it visually big in anyway? just text over white right? we have to be careful with the flex of the team name when it has spaces eg. “*Parabol Product” or the team name wraps.